src/main/java/in/projecteka/gateway/GatewayConfiguration.java

```diff
@@ -6,21 +6,7 @@
 import com.google.common.cache.CacheBuilder;
 import com.google.common.cache.CacheLoader;
 import com.google.common.cache.LoadingCache;
-import in.projecteka.gateway.clients.AdminServiceClient;
-import in.projecteka.gateway.clients.ClientErrorExceptionHandler;
-import in.projecteka.gateway.clients.ConsentFetchServiceClient;
-import in.projecteka.gateway.clients.ConsentRequestServiceClient;
-import in.projecteka.gateway.clients.DataFlowRequestServiceClient;
-import in.projecteka.gateway.clients.DiscoveryServiceClient;
-import in.projecteka.gateway.clients.HealthInfoNotificationServiceClient;
-import in.projecteka.gateway.clients.HipConsentNotifyServiceClient;
-import in.projecteka.gateway.clients.HipDataFlowServiceClient;
-import in.projecteka.gateway.clients.HiuConsentNotifyServiceClient;
-import in.projecteka.gateway.clients.IdentityProperties;
-import in.projecteka.gateway.clients.IdentityServiceClient;
-import in.projecteka.gateway.clients.LinkConfirmServiceClient;
-import in.projecteka.gateway.clients.LinkInitServiceClient;
-import in.projecteka.gateway.clients.PatientSearchServiceClient;
+import in.projecteka.gateway.clients.*;
 import in.projecteka.gateway.common.DefaultValidatedRequestAction;
```

Comment:
No No.

Reply:
:D I remove and this comes back again !!! fixing it 😂